src/napari_omero/plugins/loaders.py

```diff
@@ -78,9 +78,16 @@ def get_omero_metadata(image: ImageWrapper) -> dict:
     colors = []
     for ch in channels:
         # use current rendering settings from OMERO
-        color = ch.getColor().getRGB()
-        color = [r / 256 for r in color]
-        colors.append(Colormap([[0, 0, 0], color]))
+        color = ch.getColor().getHtml()
+        # work around a napari bug https://github.com/napari/napari/issues/7504
+        if color == "000000":
+            colors.append(ensure_colormap("gray_r"))
+        if color == "FFFFFF":
+            colors.append(ensure_colormap("gray"))
```

Comment:
Can we also handle "red", "green", "blue" and "yellow" here (and maybe "cyan" and "magenta" too)?

Reply:
We shouldn't need those because they should already be properly handled by `ensure_colormap`, e.g. magenta
```
In [1]: from napari.utils.colormaps import ensure_colormap

In [2]: ensure_colormap('#FF00FF')
Out[2]: 
Colormap(colors=ColorArray([[0., 0., 0., 1.],
            [1., 0., 1., 1.]], dtype=float32), name='magenta', interpolation=<ColormapInterpolationMode.LINEAR: 'linear'>, controls=array([0., 1.], dtype=float32))
```
Passing the hex already finds the correct colormap. It just doesn't work for black and white.

Reply:
Have you chosen a LUT in iviewer (or Preview panel of webclient) and saved the rendering settings?

Reply:
You'll get e.g. `inferno.lut` which may match a LUT from napari.
If `getLut()` returns `None` or a LUT that isn't found in napari, you can fall back to using the channel color.

Reply:
@will-moore I added a dict to handle the most common colors so they work in napari <0.4.19 and current napari.